### ISSUE TITLE
965 977 excluded?

### ISSUE THREAD

**Issue by ucaszyp** [nickgkan/beauty_detr#39]
Thank you for your work.

Could you please explain the meaning of the following code block? Why were scenes 965 and 977 excluded?

if self.split == 'train':
            annos = [
                anno for a, anno in enumerate(annos)
                if a not in {965, 977}
            ]

**Comment by nickgkan** [comment]
Hi, we had some trouble with misaligned/missing annotations so we excluded those two scenes from our training set. Note that we use all scenes of the test set so our results are still comparable to other approaches.
